src/Dfe.PlanTech.Web/Views/Pages/GetContentSupportPage.cshtml

```diff
@@ -0,0 +1 @@
+<h1>Hello</h1>
```

Comment:
👀 test page?

Reply:
Ha! Could have sworn I removed that already
[856115a](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/900/commits/856115aa950cae32090b514b9ea78f210fd8254b)